src/components/LeftDrawer/LeftDrawer.test.tsx

```diff
@@ -66,7 +66,7 @@ afterEach(() => {
 });
 
 describe('Testing Left Drawer component for SUPERADMIN', () => {
-  test('Component should be rendered properly', () => {
+  test('Component should be rendered properly', async () => {
```

Comment:
No need of applying async to this function, because this function is synchronous and no await is used within it